data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,946 @@
+			`	From above, this city has almost perfect radial symmetry, a sure sign that it was planned and built as a single unit rather than expanding at random over time. Every single building in the city is made from the same metal alloy, a dull and slightly irregular grey color. The corners of each building are slightly beveled instead of meeting at right angles.`
+			`	Even the spaceport is symmetrical: a ring of hangars face outwards at evenly spaced intervals, and warehouses and living quarters dot the inside of the ring.`
+			`	While the layout is utilitarian, it is an incredibly comfortable place to live, and the Hai have given their individual community spaces quite a lot of character and life.`
+			`	As you are wandering around and getting a feel for the place, you overhear someone talking about an apparently complete ringworld in Alta Hai. It definitely sounds like something you should check out.`
+				accept
+
+	#Goto Stop 3 Wah Yoot (Darkcloak) - must happen after Quarg first contact
+mission "ZZ Hai Tour [3] Meet Unfettered"
+	landing
+	name "Meet the Rebels on <planet>"
+	description "Go to <destination> and learn more about the Hai's conflict."
+	source "Alta Hai"
+	destination "Darkcloak"
+	to offer
+		has "Hai Tour [2] Alta Hai: done"
+	on offer
+		conversation
+			`This is a complete Quarg ringworld, the product of millennia of construction effort which has consumed every other stellar object in this system in order to provide the raw materials. The inner edge of the ring is filled with solar panels, hydroponic gardens, living quarters, and sunlit common spaces. The outer edge contains hangars and docks for ships while housing storage facilities for cargo.`
+			`	Pondering your questions, you find a Quarg and ask, "So this isn't Hai space then, but it's a dead end?"`
+			`	The Quarg explains, "Much like in Enif in human space, this is where we live as friends to the Hai."`
+			`	This explanation seems typically Quarg, but it's the best one you're able to get. Before you go, you ask, "Do you know about the rebels I've been hearing about?"`
+			`	The Quarg responds, "The Hai and their rebels are the same. You should travel to Darkcloak in Wah Yoot to understand."`
+			`	It seems like there's nothing else for it but to see for yourself.`
+				accept
+	
+	#Goto Stop 4 Wah Ki (Cloudfire)
+	# Must happen before unfettered first contact
+mission "00 Hai Tour [4] Cloudfire"
+	landing
+	name "Find out about the Rebels on Cloudfire"
+	description "Go to Cloudfire and compare the Hai's side of the conflict."
+	source "Darkcloak" 
+	destination "Cloudfire"
+	to offer
+		or
+			has "Hai Witness Unfettered [1]"
+			has "ZZ Hai Tour [3] Meet Unfettered: done"
```

Comment:
This didn't fire for me, I got the usual history lesson and request for jump drives instead. After flying around the galaxy for a while, I figured something was wrong and temporarily changed `has "Hai Witness Unfettered [1]"` to `has "Hai Witness Unfettered [1]: done"` so I could trigger this by landing on Cloudfire again.

Would it make sense to give "Hai Witness Unfettered [1]" a stopover on Darkcloak with Cloudfire as the destination and then reuse the completion conversation of this mission there as well? Hopefully that would avoid any weird interactions with the normal unfettered stuff and allow the next mission to pick up as it should.

Reply:
Congrats you appear to have found a bug that slipped through initial testing, this will be fixed.